### ISSUE TITLE
will ros2 launch mavros?

### ISSUE THREAD

**Issue by Boshen97** [PX4/px4_ros_com#125]
Hello, I have a question to consult.

Are there any plans to introduce an abstraction layer similar to the mavros function package in ROS2?

**Comment by TSC21** [comment]
@Boshen97 yes. It is in our plans to provide an API to facilitate the usage of the bridge. More info can be found on the presentations this year on the PX4 Dev Summit and ROS World.

**Comment by TSC21** [comment]
@Boshen97 also to clarify: we will have an abstraction layer that interfaces with the microRTPS bridge, not with MAVROS. px4_ros_com is completely different approach to bridge PX4 and ROS2. So nothing here interfaces or launches MAVROS.

**Comment by Boshen97** [comment]
Thanks for your reply, now I understand

**Comment by Boshen97** [comment]
When is the launch time of this abstraction layer？

**Comment by TSC21** [comment]
> When is the launch time of this abstraction layer？

No current timeline. Something to aim on the next 2/3 months.
